### PR TITLE
[bitnami/etcd] add priority class name option to etcd

### PR DESCRIPTION
**Pull request by turkenh**
**Description of the change**

This PR introduces `priorityClassName` variable to the etcd helm chart.
 
**Benefits**

Most of the time, etcd is a critical component being a persistent component and for Kubernetes, being the single persistent component. We usually don't want etcd pods to be preempted for other workloads to be scheduled. 

This PR enables setting a `priorityClassName` for etcd pods: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/

**Possible drawbacks**

None

**Applicable issues**

  - fixes #

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

File: bitnami/etcd/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: etcd
-version: 4.6.4
+version: 4.6.5
```
Comment (dani8art) [inline bitnami/etcd/Chart.yaml]: I would add a minor version since we are adding a new value

Comment (turkenh) [reply bitnami/etcd/Chart.yaml]: Agreed, done!

File: bitnami/etcd/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: etcd
-version: 4.6.4
+version: 4.7.4
```
Comment (dani8art) [inline bitnami/etcd/Chart.yaml]: minor version according to semver would be `4.7.0`

Comment (turkenh) [reply bitnami/etcd/Chart.yaml]: you're right, sorry 🤦‍♂

**Pull request merged**